File: app/src/main/java/com/penseapp/acaocontabilidade/chat/presenter/ChatsWriterPresenter.java
package com.penseapp.acaocontabilidade.chat.presenter;

/**
 * Created by unity on 21/11/16.
 */

public interface ChatsWriterPresenter {
    void createChatIfNeeded(String senderId, String senderName, String senderCompany, String recipientId, String recipientName, String recipientCompany);
    void onChatCreated(String chatId, String chatName);
}

File: app/src/main/java/com/penseapp/acaocontabilidade/chat/chats/presenter/ChatsWriterPresenter.kt
package com.penseapp.acaocontabilidade.chat.chats.presenter

/**
 * Created by unity on 21/11/16.
 */

interface ChatsWriterPresenter {

    fun createChatIfNeeded(senderId: String,
                           senderName: String,
                           senderCompany: String,
                           recipientId: String,
                           recipientName: String,
                           recipientCompany: String)

    fun onChatCreated(chatId: String, chatName: String)
}
